openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.test.tsx

```diff
@@ -79,7 +79,7 @@ describe('Test Explore component', () => {
     );
     const searchData = await findByTestId(container, 'search-data');
     const wrappedContent = await findByTestId(container, 'wrapped-content');
-    const tabs = await findAllByTestId(container, /tab/i);
+    const tabs = await findAllByTestId(container, /-tab/i);
```

Comment:
@aniketkatkar97 , Why we are changing this?

Reply:
So as I changed datatestid for search-dropdowns changed from 'search-dropdown' to 'search-dropdown-${dropdownName}', `/tab/i` was also returning the div for database dropdown while fetching tabs divs.

Reply:
@aniketkatkar97, Please add comments for such changes.